### ISSUE TITLE
Resistance: Fall of Man - Closes

### ISSUE THREAD

**Issue by Jonathan44062** [RPCS3/rpcs3#11435]
In the chapter "A Desperate Gamble" the game itseft closes without any explanation, It's not VRAM issue, even using FRS Upscaling, the game is still crashing, deleting shader cache, ppu and spu cache doesn't work 

_Please attach TWO logs:_
One for the build with regression.
[RPCS3.log](https://github.com/RPCS3/rpcs3/files/7922402/RPCS3.log) 0.0.20-13155 (#11335) <- Regression
One for the build that works as expected.
[RPCS3.log.gz](https://github.com/RPCS3/rpcs3/files/7922703/RPCS3.log.gz) 0.0.20-13151 (#11345) <- Last build working fine

_Please attach screenshots of your problem._
Here's an video how this issue happen, and there's no error message in the log

https://user-images.githubusercontent.com/78843189/150712409-f1655801-69bb-4bd9-a634-fc7ad9b9d8f0.mp4

_Please provide your system configuration:_
OS - Windows 10 21H2 Build: 19044
CPU - i7 8700
GPU - GTX NIVIDIA 1070 Ti
Driver version - 511.23.0.0

**Comment by kd-11** [comment]
Please provide a save file for this one.

**Comment by Jonathan44062** [comment]
https://www.mediafire.com/file/ssm43xz4ribh2fs/Resistance_Fall_of_Man_Save_Data_US.rar/file
For some reason, I can't drop the save file here, here's the link

**Comment by SolomonGrundyThanksForTheTurkey** [comment]
Funny thing is i had reported this as an issue last year. Interestingly enough, the game seems to not freeze if you just brute past that part. If you sit in that are for a little it will crash... after you checkpoint it will not crash and you can return to that area with no issues. I've beaten the entire game from start to finish and that was the only issue i had - but it resolved itself , or rather i resolved it by spending as little time as possible by that staircase.

**Comment by SolomonGrundyThanksForTheTurkey** [comment]
TLDR: Just walk past the enemies and fight them at the bottom of the staircase rather than at the start of it. should work.

**Comment by Jonathan44062** [comment]
It doesn't work for me, I tried like 4 times and the game closes, even running faster that area, the game still crashes and close the emulator

**Comment by Meriadock88** [comment]
Same issue for me! But game can crashes at any area of this level!
And this is not a VRAM cause I see that it not over 500-600mb of my 4Gb videocard.

Also can confirm that this issue appear not only in this level but in "Viper's Nest" to! "Viper's Nest" I played with OpenGL and this helped to beat level.
But in "A Desperate Gamble" this issue more often and you can't progress game cause of this no matter Vulkan or OpenGL!

I tried lower resolution at minimum.
Tried OpenGL and Vulkan.
Tried play with/without game patches.
Vsync ON/OFF

**Comment by Meriadock88** [comment]
So...
I did some research!
In forum I find replies from people whom complete the game without crashind at this area! Those I seen are dated in period of 08.2021 so I down;oaded rpcs3 for that period (rpcs3-v0.0.17-12695-ae491f04_win64)

And guess what? I passed this level without crashes at all!
Maybe this info helps for somehow to stabilisate this game at future updates!

**Comment by SolomonGrundyThanksForTheTurkey** [comment]
Makes sense! That’s around the period in which I completed it!

**Comment by Jonathan44062** [comment]
I'm gonna to share a new safe file, since I completed the game in the build 0.0.17-12695 (Like Meriadock88 mentioned), the game has a weird way to save the progress, now kd-11 can analyze the issue, the game closes in the chapter A Desperate Gamble, in London, new link:

https://www.mediafire.com/file/j77np03j0wn9lc8/Resistance_Fall_of_Man_Save_Data_US_NEW.rar/file
